pkg/logs/util/eventlog/transform.go

```diff
@@ -3,87 +3,135 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
 // Copyright 2023-present Datadog, Inc.
 
-//go:build windows
-
-package windowsevent
+// Package eventlog contains utilities to transform Windows Event Log XML messages into structured messages for Datadog Logs.
+package eventlog
```

Comment:
There are many types of events and logs in the agent. Would it be possible to change the name to be more specific, for example `windowsevent` or `windowseventlog`?